schema/google/showcase/v1beta1/echo.proto

```diff
@@ -222,6 +222,9 @@ message EchoRequest {
   optional string other_request_id = 8 [
     (google.api.field_info).format = UUID4
   ];
+
+  // Http request headers to echo via EchoResponse.http_request_header_value
```

Comment:
I am not sure we need a field for this, I think sending everything that appears in the request back is fine.

Reply:
Fixed in https://github.com/googleapis/gapic-showcase/pull/1501/commits/51151734e80fb6eb15a7d88c09a8c7a080e53ff0

Updated response based on local testing:
```
response
content: "The hail in Wales falls mainly on the snails."
request_id: "some_value"
http_request_header_value {
  key: "x-goog-api-version"
  value {
    header_values: "v1_20240408"
  }
}
http_request_header_value {
  key: "x-goog-api-client"
  value {
    header_values: "gl-python/3.9.16 grpc/1.63.0.dev0 gax/2.18.0 gapic/0.0.0"
  }
}
http_request_header_value {
  key: "user-agent"
  value {
    header_values: "grpc-python/1.63.0.dev0 grpc-c/39.0.0 (linux; chttp2)"
  }
}
http_request_header_value {
  key: "grpc-accept-encoding"
  value {
    header_values: "identity, deflate, gzip"
  }
}
http_request_header_value {
  key: "content-type"
  value {
    header_values: "application/grpc"
  }
}
http_request_header_value {
  key: ":authority"
  value {
    header_values: "localhost:7469"
  }
}
```